### PR TITLE
Fix missing name for inner deployment object

### PR DESCRIPTION
**Pull request by pcasaretto**
When hacking on heaven I noticed this small mistake.

### CODE REVIEW

**Comment by SSYouknowit** [comment]
Going once

File: lib/webhooks/deployment.payload.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "sha": "69a8b72e2d3d955075d47f03d902929dcaf74033",
     "ref": "master",
     "task": "deploy",
+    "name": "baxterthehacker/public-repo",
```
Comment (jasonrudolph) [inline lib/webhooks/deployment.payload.json]: @pcasaretto: I checked our current production webhook payloads for deployment events, and I'm not seeing this attribute. Can you share a recent example that shows that this attribute is included in the payloads you're seeing?

Comment (pcasaretto) [inline lib/webhooks/deployment.payload.json]: Well, to tell you the truth I just assumed it was.
Heaven's test_fixtures had it and all the other attributes were replicated.
I just checked a recent one from a repo of mine and it's not actually there.
Now I'm confused. Whats the difference between the top level attributes and the inner objects?

File: lib/webhooks/deployment.payload.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "sha": "69a8b72e2d3d955075d47f03d902929dcaf74033",
```
Comment (viethoang1102) [inline lib/webhooks/deployment.payload.json]: Viethoang

File: lib/webhooks/deployment.payload.json (hunk 1 above)
Comment (jasonrudolph) [inline lib/webhooks/deployment.payload.json]: > Whats the difference between the top level attributes and the inner objects?

During the preview period for the Deployments API, the payloads originally had those top-level attributes, but they were removed prior to shipping the final (production) version of the Deployments API. This [blog post](https://developer.github.com/changes/2014-10-21-deployment-webhook-payload-changes/) describes the changes.

Heaven's test fixtures are likely using the old payload format.

I've opened an internal issue for us to update these example payloads to reflect current production functionality. Thanks for helping us spot this discrepancy. :bow:

**Comment by jasonrudolph** [comment]
@pcasaretto: Following up on https://github.com/github/developer.github.com/pull/762#discussion_r29589525, we've updated the example payloads (via https://github.com/github/developer.github.com/pull/766) to reflect current production functionality. You can see the updated deployment payloads at https://developer.github.com/v3/activity/events/types/#deploymentevent.

Now that those payloads are updated, I'm going to close this PR. Thanks again for helping us spot this discrepancy. :cake:

**Comment by pcasaretto** [comment]
Thank you for promptly addressing this! :beers:

**Comment by pcasaretto** [comment]
@jasonrudolph I might be tired from a long day's work but something seems to be off.
[This guy](https://github.com/github/developer.github.com/blob/9c97519f8cf8963305104fe18c8f67fef20295fb/content/v3/activity/events/types.md#deploymentevent) mentions a `name` key in the payload.
While [this guy](https://github.com/github/developer.github.com/blob/master/lib/webhooks/deployment.payload.json) only has a `name` key under the `repository` object.
Makes sense?

**Comment by jasonrudolph** [comment]
@pcasaretto: Thanks for pointing out that issue. It should be resolved as of https://github.com/github/developer.github.com/pull/772. If you notice any lingering inaccuracies, please let us know. :bow:

**Comment by pcasaretto** [comment]
Happy to help.

**Pull request closed without merging**